src/background/providers/cloudflare.test.ts

```diff
@@ -0,0 +1,334 @@
+import { jest } from '@jest/globals';
+import { CloudflareProvider } from './cloudflare';
+import { Storage } from '../storage';
+import Token from '../token';
+
+export class StorageMock {
+    store: { [key: string]: string };
+
+    constructor() {
+        this.store = {};
+    }
+
+    getItem(key: string): string | null {
+        if (key in this.store) {
+            return this.store[key];
+        } else {
+            return null;
+        }
+    }
+
+    setItem(key: string, value: string): void {
+        this.store[key] = value;
+    }
+}
+
+test('getStoredTokens', () => {
+    // @ts-ignore
+    const storage: Storage = new StorageMock();
+    const updateIcon = jest.fn();
+    const navigateUrl = jest.fn();
+
+    const provider = new CloudflareProvider(storage, { updateIcon, navigateUrl });
+    const tokens = [new Token(), new Token()];
+    // @ts-ignore
+    provider.setStoredTokens(tokens);
```

Comment:
The method is private, so I think is safer to use the index notation method rather than remove the compilers check with `@ts-ignore`.  

```suggestion
    provider["setStoredTokens"](tokens);
```